code/dynamic_programming/Array_Median/median.cpp

```diff
@@ -0,0 +1,21 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+int main()
+{
+    int n,median;
+    cout<<"Enter number of elements\n";
+    cin>>n;
+    vector<int> arr(n);
+    for(int i=0;i<n;i++){
```

Comment:
If using vectors, don't ask for size, and allow user to enter elements until a EOF. (Use `vector::push_back`)